tools/twix/src/panels/map/layers/behavior_simulator.rs

```diff
@@ -61,7 +64,7 @@ impl Layer<Field> for BehaviorSimulator {
         painter: &TwixPainter<Field>,
         _field_dimensions: &FieldDimensions,
     ) -> Result<()> {
-        for (player_number, player_handle) in self.ground_to_field.0.iter() {
+        for (jersey_number, player_handle) in &self.ground_to_field.0.inner {
```

Comment:
The player_handle is a BufferHandle, calling it a jersey_handle feels wrong because it is not referring to the jersey, but rather the player wearing that jersey. What do you suggest?

Reply:
but why is the iterable called ground_to_field?

Reply:
Because it gives access to the ground to field transformation for all robots.